src/mbgl/text/placement.cpp

```diff
@@ -253,12 +258,12 @@ void Placement::placeBucket(
                             textBoxScale,
                             prevAnchor
                         }));
-                        markUsedJustification(bucket, anchor, symbolInstance);
```

Comment:
Do we have tests for auto justification + variable anchors? What is the reason for removal?

Reply:
we have few render tests, for example `text-variable-anchor/all-anchors`. These calls are not needed here at they are duplicated inside `Placement::updateBucketOpacities()`